src/aks-preview/azext_aks_preview/decorator.py

```diff
@@ -1485,7 +1539,7 @@ def construct_preview_mc_profile(self) -> ManagedCluster:
         mc = self.set_up_pod_identity_profile(mc)
         return mc
 
-    def create_mc(self, mc: ManagedCluster) -> ManagedCluster:
+    def create_preview_mc(self, mc: ManagedCluster) -> ManagedCluster:
```

Comment:
create_mc_preview or

preview_create_mc

Reply:
Yep, changed in commit [#29af4f4](https://github.com/Azure/azure-cli-extensions/pull/4185/commits/29af4f44579f5bf893e15aacf925320e558f100b).